stats.go

```diff
@@ -312,6 +323,20 @@ func newPoint(v *view.View, row *view.Row, start, end time.Time) *monitoringpb.P
 	}
 }
 
+func newGaugePoint(v *view.View, row *view.Row, end time.Time) *monitoringpb.Point {
+	gaugeTime := &timestamp.Timestamp{
+		Seconds: end.Unix(),
+		Nanos:   int32(end.Nanosecond()),
+	}
+	return &monitoringpb.Point{
+		Interval: &monitoringpb.TimeInterval{
+			StartTime: gaugeTime,
```

Comment:
As I recall `StartTime` is optional for `Gauge` values. You can safely remove this field for `GaugePoint`.

See https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeInterval.

Reply:
oh sorry I miss your comments before I merge. I would revise it in the future PR ( If there is :)

Reply:
No worries :) Since the StartTime needs to be the same as EndTime anyway, I thought it might be easier to just remove it and have it used the default value.